datasets/peer_read/peer_read.py

```diff
@@ -0,0 +1,289 @@
+                    "accepted": datasets.Value("bool"),
+                    "abstract": datasets.Value("string"),
+                    "histories": datasets.features.Sequence(
+                        datasets.features.Sequence(datasets.Value("string"))
+                    ),
+                    "reviews": datasets.features.Sequence({
+                        "date": datasets.Value("string"),
+                        "title": datasets.Value("string"),
+                        "other_keys": datasets.Value("string"),
+                        "originality": datasets.Value("string"),
+                        "comments": datasets.Value("string"),
+                        "is_meta_review": datasets.Value("bool"),
+                        "is_annotated": datasets.Value("bool"),
+                        "recommendation": datasets.Value("string"),
+                        "replicability": datasets.Value("string"),
+                        "presentation_format": datasets.Value("string"),
+                        "clarity": datasets.Value("string"),
+                        "meaningful_comparison": datasets.Value("string"),
+                        "substance": datasets.Value("string"),
+                        "reviewer_confidence": datasets.Value("string"),
+                        "soundness_correctness": datasets.Value("string"),
+                        "appropriateness": datasets.Value("string"),
+                        "impact": datasets.Value("string"),
+                    })
+                }
+            )
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=features,  # Here we define them above because they are different between the two configurations
+            # If there's a common (input, target) tuple from the features,
+            # specify them here. They'll be used if as_supervised=True in
+            # builder.as_dataset.
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage=_HOMEPAGE,
+            # License for the dataset if available
+            license=_LICENSE,
+            # Citation for the dataset
+            citation=_CITATION,
+        )
+
+    @staticmethod
+    def _get_paths(data_dir, domain):
+        paths = {
+            'train': [], 'test': [], 'dev': [],
+        }
+        parent_dirpth, conferences, _ = next(os.walk(f'{data_dir}/PeerRead-master/data'))
+        for conference in conferences:
+            dirpth, data_types, _ = next(os.walk(f'{parent_dirpth}/{conference}'))
+            for data_type in data_types:
+                for sub_dirpth, _, fnames in sorted(os.walk(f'{dirpth}/{data_type}/{domain}')):
+                    for fname in sorted(fnames):
+                        paths[data_type].append(f'{sub_dirpth}/{fname}')
```

Comment:
can you use os.path.join instead of concatenating the paths with '/' ? 

Also can you apply sorted to each one of the paths lists (train, test, dev) before returning the `paths` dict to make the order deterministic ?